Adding chart titles from specifications

alexantoinefortin opened the pull request:
* each charts gets its own title
* titles are **bolded** and accounts for plural
* xaxis title for `infection_graph()` is more clear (`Number of days from today` vs `Days`)

Comments / feedback  are welcomed!

Thanks!

archydeberker commented:
Great! Thanks. Could you possibly: 
- make sure the titles are all the same size
- optional: change the font to match the streamlit font (this is a bit demanding, sorry - the Plotly graphs have always had a different font and it just wasn't obvious because there was so little tex in them). I'm happy to do this in a followup PR

alexantoinefortin commented:
* make sure the titles are all the same size
fixed in 7d87ad1

* optional: change the font to match the streamlit font (this is a bit demanding, sorry - the Plotly graphs have always had a different font and it just wasn't obvious because there was so little tex in them). I'm happy to do this in a followup PR

Fixed in 481ec0c. I inspected the webpage and found that 2 fonts were used: `Open Sans` and `Arial`. When looking at the font, I found that `Arial` is more aligned with the webpage's text. 

let me know if that works well for you.

alexantoinefortin commented:
(somebody with write access to this repository must perform the merge)
